.vale/styles/style_guide/Headings.yml

```diff
@@ -48,4 +49,8 @@ exceptions:
   - App Service Auto-Scale
   - Certbot
   - Bleve & Tantivy
-  - '([0-9a-z]:\s[A-Z])'  #How do get this to throw an error for `Step 1: do this`?
+  - Google Cloud Console
+  - AMIs
+  - '\b\s[0-9]:\s[A-Z]' # Step 1: Do this
+  - '[0-9]\.\s[A-Z]'  # 1. Do this
```

Comment:
The two comments are describing the correct capitalisation in these scenarios, right?

Reply:
Yes, the first one is for headings that start with `Step X:` and the second one is for headings that are numbered lists